CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 project(co2mon)
 cmake_minimum_required(VERSION 2.8)
 
-set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -Wall -Wextra -Wno-unused-parameter")
```

Comment:
Because `-Wno-unused-parameter` suppresses all warnings(not only false-positive). To suppress a warning you force compiler with macro, which means you know what you are doing.